docs/source/tutorials/outliers.ipynb

```diff
@@ -0,0 +1,627 @@
+    "axes[1].set(title='test_outlier_scores distribution', ylabel='Frequency')\n",
+    "axes[1].axvline(x=fifth_percentile, color='red', linewidth=2)\n",
+    "\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "55362e5d",
+   "metadata": {},
+   "source": [
+    "Everything to the left of the red line in the distribution of `test_outlier_scores` will be marked as an outlier. Let's measure how well this works. \n",
+    "\n",
+    "**Visually check the threshold**\n",
+    "\n",
+    "First lets plot the least sure outliers of our `test_data`. These are the images immediately to the left of that cuttoff line. As you can see majority of them are still true outliers however there are a few less standard looking animals that are now falseley identified as outliers as well."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "b2e40fea",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Visualize 15 outlier scores right along the cuttoff (i.e. the least likeley outliers given threshold)\n",
+    "sorted_idxs = test_outlier_scores.argsort()\n",
+    "outlier_scores = test_outlier_scores[sorted_idxs]\n",
+    "outlier_indices = sorted_idxs[outlier_scores < fifth_percentile] # Images in test data flagged as outliers\n",
+    "\n",
+    "selected_outlier_subset = torch.utils.data.Subset(test_data, outlier_indices[::-1])\n",
+    "plot_images(selected_outlier_subset)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "eb8c74f4",
+   "metadata": {},
+   "source": [
+    "**Empirically measure threshold effectiveness**\n",
+    "\n",
+    "Setting the hard threshold to the 5-th percentile of the `train_data` gives us almost exactly a 5% false positive rate. If that is what we are looking for, this is an effective threshold cuttoff for this data distribution."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "24a40cba",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "animal_classes = [2,3,4,5,6,7] # Animal classes in the dataset\n",
+    "test_data_labels = np.array(test_data.dataset.targets) # Get subset labels from out test_data\n",
+    "animal_idxs = np.where(np.isin(test_data_labels[test_data.indices], animal_classes))[0]\n",
+    "\n",
+    "false_positive_idxs = set(outlier_indices).intersection(set(animal_idxs))\n",
+    "FPR = len(false_positive_idxs) / (len(false_positive_idxs) + len(animal_idxs))\n",
+    "print(f'Number of false positives detected: {len(false_positive_idxs)}\\nFalse positive rate: {round(FPR,4)} ')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "740a8774",
+   "metadata": {
+    "nbsphinx": "hidden"
+   },
+   "outputs": [],
+   "source": [
+    "# Note: This cell is only for docs.cleanlab.ai, if running on local Jupyter or Colab, please ignore it.\n",
+    "num_animals = 0\n",
+    "for i in range(15):\n",
+    "    _,y = top_outlier_subset[0]\n",
```

Comment:
just FYI there's small bug here I believe; should be `i` not `0` right?

Reply:
Also just FYI it's typically not a good idea to hardcode a number like 15 as done here. 
Instead all 15s would be replaced by: `len(top_outlier_subset)`

Reply:
Don't worry about these though, I'll make the updates in my future PR editing this notebook

Reply:
Oh yeah i should be the value! 
I hardcoded 15 since 15 images are shown for the top outliers so it would make sense to make sure we're at least visualizing good data but len(top_outlier_subset) works the same for this case.